components/Bounty/api/fetchOpenBounties.ts

```diff
@@ -17,13 +24,21 @@ export type SimpleBounty = {
   expiration_date: string;
   id: ID;
   item: any /* intentional any */;
+  plainText: string;
+  unifiedDocument: UnifiedDocument;
 };
 
-export const fetchOpenBounties = ({ onError, onSuccess, page }: Args): void => {
+export const fetchOpenBounties = ({
+  onError,
+  onSuccess,
+  model,
+  page,
+}: Args): void => {
   fetch(
     buildApiUri({
       apiPath: "bounty/get_bounties",
-      queryString: "?status=OPEN",
+      queryString:
+        "?status=OPEN" + (model ? `&item_content_type__model=${model}` : ""),
```

Comment:
model is an optional way to filter specific types of comments. Depending on the bounty type (i.e. comment bounty or not), parsing of it happens differently. 

There are three options:
1. Update the BE to always return comment bounties only
2. Update the FE to filter for only comment bounties
3. Update BE to support filtering for specific bounty types

3 seemed like the best option because we will likely have other bounty types in the future and also we have historical bounties that are not all comment bounties so introducing filtering capability seemed the best. LMK if you have other thoughts/suggestion